src/components/QuickStart/index.jsx

```diff
@@ -166,6 +166,14 @@ start()`
   return (
     <>
       <h1>Quick start</h1>
+
+      <a
+        href="https://gitpod.io/#https://github.com/gitpod-samples/template-fastify"
```

Comment:
I would prefer for the template to be sitting in the Fastify org, so we can change/modify etc.

Reply:
If you could create one I can change it out!

Reply:
Can you add the necessary files in https://github.com/fastify/fastify-template-gitpod?

Reply:
I'm sorry for the late reply @mcollina I've only just seen this :sweat_smile: - I've updated the link on the docs here and made a PR to the template: https://github.com/fastify/fastify-template-gitpod/pull/1